### PR TITLE
[env/github] replace sparkle logo with github logo

### PR DESCRIPTION
**Pull request by malaikaaiyar**
partially fixes https://github.com/sparkletown/internal-sparkle-issues/issues/821

Screen Shot 2021-06-25 at 10 34 59 PM

### CODE REVIEW

**Comment by whoaitsaimz** [comment]
TBD -- please await @sparkleed decision on whether we will do this

**Comment by malaikaaiyar** [comment]
@sparkleed has decided that this is good to go!

File: src/pages/Account/Account.scss [hunk 1]
```diff
@@ -120,6 +120,16 @@ $page-max-width: 1240px;
...
+
+  &.github-plain {
+    margin: 0 auto;
+    background-image: url(/assets/images/Summit-logo-plain.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 580px;
+    height: 225px;
+    background-size: 400px;
+  }
```
Comment (0xdevalias) [inline src/pages/Account/Account.scss:132]: Does the old `/assets/images/Summit_Logo.png` still exist here? And does it need to? Is it used anywhere else? Or was it intended for this location/use case?

File: src/pages/Account/Login.tsx [hunk 2]
```diff
@@ -87,9 +87,7 @@ export const Login: React.FC<LoginProps> = ({
 
   return (
     <div className="auth-container">
-      <div className="logo-container">
-        <img src="/sparkle-header.png" alt="" width="100%" />
-      </div>
+      <div className="hero-logo github-plain"></div>
```
Comment (0xdevalias) [inline src/pages/Account/Login.tsx:90]: HTML tags without content in them should use the shorthand tag syntax:
```suggestion
      <div className="hero-logo github-plain" />
```

File: src/pages/Account/Account.scss (hunk 1 above)
Comment (0xdevalias) [reply src/pages/Account/Account.scss:132]: Seems it's used in `src/pages/Account/Questions.tsx`

```
<div className="hero-logo github"></div>
```

**Review by 0xdevalias — APPROVED** [review]
Looks good, LGTM! :shipit: 

---

**FYI**

Optimised the logos with imageoptim https://github.com/sparkletown/sparkle/pull/1666/commits/b9913ed41c64cd8fd24d354d8459c822c57423ee

image

## Screenshots

**Initial login page:**
image

**Log in for non-Hubbers page:**
image

**Pull request merged**